packages/next/build/webpack/loaders/next-client-pages-loader.ts

```diff
@@ -21,7 +21,14 @@ const nextClientPagesLoader: loader.Loader = function() {
           next.router.update(${stringifiedPage}, updatedPage.default || updatedPage)
         })
       }
-      return { page: page.default || page }
+      var pageExports = {};
+      for (var exportKey in page) {
+        pageExports[exportKey] = page[exportKey];
```

Comment:
Why is this loop needed?

Reply:
@prateekbh I think we could change `page` to `mod` (module) and then you don't need this in particular 👍

Reply:
done

Reply:
So this bit of code can be removed then right?

Reply:
yes. Sorry forgot to remove the loop